### ISSUE TITLE
Elementor Menu Accessibility Issue (WCAG 2.1.1 Keyboard)

### ISSUE THREAD

**Issue by ph-webetc** [elementor/elementor#25753]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

The Elementor 'WordPress Menu' fails the WCAG 2.1.1 test for accessibility (visual impairment and keyboard interface) when a submenu is present. This was picked up in a statutory accessibility audit.

On a desktop/laptop, when a user zooms their browser to a point where the menu falls to a hamburger, they cannot tab to any items within a submenu.

As this is a critical requirement for our users, we are looking to implement a third-party menu plugin as an interim solution.

### Steps to reproduce

1. Create a menu in WordPress incorporating a submenu element
2. In Elementor, add the Wordpress Menu widget to the page, selecting the menu created above
3. Preview the site - zoom your browser to between +200 and +400% (until the menu falls to a hamburger)
5. Using the tab key, tab through until the dropdown element in the menu is in focus
6. Press enter to open the submenu
7. Tab to select an item in the submenu

At this point, the submenu will close and the focus will move to the next top-level menu item.

### Expected behavior

1. When tabbing to a submenu and pressing 'enter', the next tab press should take the user to the first item of the submenu
2. Subsequent tab presses should move the focus to the next submenu items
3. After the last submenu item, the focus should move to the next top-level menu item

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.17
	PHP Version: 7.4.33
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 8000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.2
	Site URL: https://REDACT
	Home URL: https://REDACT
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%category%/%postname%/
	Language: en_GB
	Timezone: Europe/London
	Admin Email: REDACT
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.0.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36


== Elementor Experiments ==
	Improved asset loading: Active by default
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Inactive by default
	Optimized Gutenberg Loading: Active by default
	Build with AI: Active by default
	Landing pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Optimize Image Loading: Inactive
	Notes: Inactive
	Display Conditions: Inactive by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Element Pack Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by Daveden2** [comment]
I gave up on the Elementor WordPress menu a long time ago. I'm just here to observe the responses.

**Comment by portrayaloflife** [comment]
+1

**Comment by nicholaszein** [comment]
Hello @ph-webetc! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by veryaca** [comment]
+1. Following.

**Comment by githubtoh** [comment]
Any ETA on when this will be fixed? Thanks.
